### ISSUE TITLE
UI: Make container steps collapsible

### ISSUE THREAD

**Issue by flosell** [flosell/lambdacd#59]
Complex pipelines with lots of steps of details often get very long: 
For example: 

* Build
* Run five different types of test
* Deploy to three environments, for each
  * check that your can deploy
  * deploy 
  * run smoke tests

Would be great to be able to collapse this pipeline so that initially, it's just "build", "test", "deploy env1" "deploy env2" "deploy env3" and users can drill expand to get the details

**Comment by flosell** [comment]
Now everything is collapsed per default: 

<img width="740" alt="screen shot 2015-11-21 at 16 42 18" src="https://cloud.githubusercontent.com/assets/540525/11319193/dbbb0884-906e-11e5-8803-5a4532be1181.png">

Would probably solve problems with large pipelines but it's not really obvious what happens. Therefore, a couple of more tasks to make this work: 

* [x] make steps collapsible
* [x] make steps expandable
* [ ] implement aliases
* [ ] add 'expand-all' feature (could be a bar above the pipeline)
* [ ] make collapse-behavior user-definable: e.g. you deploy to ci and qa in parallel. the `in-parallel` parent isn't the interesting part so we want it to be expanded by default

**Comment by flosell** [comment]
Added aliasing: 
<img width="707" alt="screen shot 2015-11-22 at 12 20 54" src="https://cloud.githubusercontent.com/assets/540525/11323474/890f488e-9113-11e5-8024-8e28d9f82979.png">

Looks good enough for now, the rest of the tasks seem like nice to haves, decide on them after gathering some feedback

**Comment by flosell** [comment]
Received good feedback on the current state, leaving as is for now. User definable expand/collapse behavior might still be nice to have so this is split out into a separate issue.
